### PR TITLE
Adding a EKS Purge Job, removing the AKS job, making the frequency once a day for test job

### PR DESCRIPTION
**Pull request by ytimocin**
* Adding a job that will purge AWS EKS clusters once every day at 7 AM. The reasoning behind this is that the scheduled test job will run at 12 PM and the EKS Cluster will be already 7 hours old at the time purge job runs. So, if the cluster is not deleted, this job will clean the cluster and all of its underlying resources.
* Removing the job that runs tests on the AKS Cluster.
* Decreasing the schedule to once a day.

### CODE REVIEW

File: .github/scripts/purge-aws-rds-snapshots.sh
Comment (ytimocin) [inline .github/scripts/purge-aws-rds-snapshots.sh:1]: reformat

File: .github/workflows/purge-aws-rds-snapshots.yaml
Comment (ytimocin) [inline .github/workflows/purge-aws-rds-snapshots.yaml:1]: reformat

**Review by willdavsmith — APPROVED** [review]

File: .github/scripts/purge-aws-eks-clusters.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    age=$((current_time - created_at_seconds))
+
+    # Check if age is greater than age limit and name starts with 'eks-samplestest-'
+    if [ "$age" -gt "$age_limit" ] && [[ "$name" == eks-samplestest-* ]]; then
+        echo "Deleting cluster $name older than 6 hours."
+        eksctl delete cluster --name "$name" --wait --force
+    else
+        echo "Cluster $name is not older than 6 hours or does not meet naming criteria."
+    fi
+done
```
Comment (rynowak) [inline .github/scripts/purge-aws-eks-clusters.sh:47]: very cool 👍

**Review by rynowak — APPROVED** [review]
:shipit:

**Pull request merged**